ci/scripts/check-npm-dependencies.js

```diff
@@ -48,58 +40,55 @@ const checkPackageDependencies = packageName => {
         const packResultRaw = child_process.spawnSync('npm', ['pack', '--dry-run', '--json'], {
             encoding: 'utf-8',
             cwd: PACKAGE_PATH,
         }).stdout;
 
         const packResultJSON = JSON.parse(packResultRaw);
-
         const localChecksum = packResultJSON[0].shasum;
 
         // check remote package
-        const viewResultRaw = child_process.spawnSync('npm', ['view', '--json'], {
-            encoding: 'utf-8',
-            cwd: PACKAGE_PATH,
-        }).stdout;
+        const { stderr, stdout: viewResultRaw } = child_process.spawnSync(
+            'npm',
+            ['view', '--json'],
+            {
+                encoding: 'utf-8',
+                cwd: PACKAGE_PATH,
+            },
+        );
+
+        if (stderr) {
+            errors.push(name);
+            return;
+        }
 
-        // means that this package is new and has never been released
-        if (!viewResultRaw) {
-            updateNeeded.push(dependency);
-        } else {
-            const viewResultJSON = JSON.parse(viewResultRaw);
+        const viewResultJSON = JSON.parse(viewResultRaw);
+        if (viewResultJSON.error) {
+            return;
+        }
 
-            if (viewResultJSON.error) {
-                console.log(viewResultJSON);
-                return;
-            }
+        const remoteChecksum = viewResultJSON[dependency].dist.shasum;
 
-            const remoteChecksum = viewResultJSON[dependency].dist.shasum;
-
-            if (localChecksum !== remoteChecksum) {
-                // if the checked dependency is already in the array, remove it and push it to the end of array
-                // this way, the final array should be sorted in order in which that dependencies listed there
-                // should be released from the last to the first
-                const index = updateNeeded.findIndex(lib => lib === dependency);
-                if (index > -1) {
-                    updateNeeded.splice(index, 1);
-                }
-                updateNeeded.push(dependency);
+        if (localChecksum !== remoteChecksum) {
+            // if the checked dependency is already in the array, remove it and push it to the end of array
+            // this way, the final array should be sorted in order in which that dependencies listed there
+            // should be released from the last to the first
+            const index = updateNeeded.findIndex(lib => lib === dependency);
+            if (index > -1) {
+                updateNeeded.splice(index, 1);
             }
+            updateNeeded.push(dependency);
         }
 
         checkPackageDependencies(name);
     });
-};
 
-checkPackageDependencies(packageName);
-
-const formattedDeps = updateNeeded
-    .map(dep => {
-        return dep.replace('@trezor/', '');
-    })
-    .join(',');
+    return {
+        update: updateNeeded,
+        errors,
+    };
+};
 
-process.stdout.write(formattedDeps);
+module.exports = { checkPackageDependencies };
```

Comment:
Now this file only export the `checkPackageDependencies` function but it does not call it. So probably it does not  make sense to call the file as `node ci/scripts/check-npm-dependencies connect` or `node ci/scripts/check-npm-dependencies connect-web` anymore.

And we have it documented:
* https://github.com/trezor/trezor-suite/blob/connect-release-script/docs/releases/connect-release.md#L6

And used:
* https://github.com/trezor/trezor-suite/blob/connect-release-script/.github/workflows/connect-version_check.yml#L45

Reply:
> https://github.com/trezor/trezor-suite/blob/connect-release-script/.github/workflows/connect-version_check.yml#L45

you are right, this shoudl be removed 

> https://github.com/trezor/trezor-suite/blob/connect-release-script/docs/releases/connect-release.md#L6 

and this should be rephrased.

Reply:
I created issue https://github.com/trezor/trezor-suite/issues/8537 to follow this up of this. This PR looks nice and can be merged.